mp4/stsc.go

```diff
@@ -49,6 +49,7 @@ func DecodeStscSR(hdr BoxHeader, startPos uint64, sr bits.SliceReader) (Box, err
 	b := StscBox{
 		Version: byte(versionAndFlags >> 24),
 		Flags:   versionAndFlags & flagsMask,
+		Entries: make([]StscEntry, entryCount),
```

Comment:
This line should not be here, but instead be allocated after the check of size (which it actually is). This is the cause of many problems. Just drop this line.

Reply:
You're right. That was totally an oversight. I've pushed with that removed.